Airports/EGNH/Ground Map/Regions

```diff
@@ -0,0 +1,257 @@
+; - Terminal
```

Comment:
Why are there two Regions files? You don't need anything in the Ground Map folder if you have made an SMR.